root/etc/cont-init.d/50-config

```diff
@@ -29,8 +29,12 @@ chown -R abc:abc \
     /config \
     /data
 
+<<<<<<< Updated upstream
 exec \
     s6-setuidgid abc /usr/bin/python3 ./src/manage.py migrate
+=======
+s6-setuidgid abc /usr/bin/python3 ./src/manage.py migrate
+>>>>>>> Stashed changes
```

Comment:
This looks like something's gone a bit wrong with your git stashing

Reply:
d'oh!